scripts/images.sh

```diff
@@ -17,7 +17,8 @@ function pull_images() {
     for project in ${PROJECTS[*]}; do
         clone_repo
         checkout_project_branch
-        local project_version=$(_git describe --tags --exclude="${CUTTING_EDGE}" --exclude="subctl-devel" --exclude="latest")
+        local base_branch="${release['branch']:-devel}}"
+        local project_version=$(base_branch)-$(shell git rev-parse --short HEAD)
```

Comment:
```suggestion
        local project_version=$(base_branch)-$(_git rev-parse --short HEAD)
```

This should work, however I think it'd be simpler to have a simple `make print-version` command in Shipyard which'll just print this info, I'll send a patch

Reply:
We have to make a release today :-)
we can continue with that. let me fix this.

Reply:
I agree that the print-version will be better long term :+1: but hopefully this let's us start the release

Reply:
https://github.com/submariner-io/shipyard/pull/524/files